libcaf_core/caf/uri.test.cpp

```diff
@@ -314,6 +314,19 @@ TEST("from string") {
   ROUNDTRIP("http://me@node:80?a=1&b=2");
   ROUNDTRIP("http://me@node:80#42");
   ROUNDTRIP("http://me@node:80?a=1&b=2#42");
+  ROUNDTRIP("http://user:@node");
```

Comment:
These are also valid URIs (in case there's no equivalent yet):

```
  ROUNDTRIP("http://@node");
  ROUNDTRIP("http://:@node");
```

Reply:
Both of these failed. Fixed the implementaiton.